aries-site/src/pages/components/tooltip.mdx

```diff
@@ -34,6 +34,12 @@ Make sure to test Tooltip positioning to ensure that the content does not block
 Tooltips should support both mouse hover and keyboard focus.
 Do not use tooltips for information that is vital for the user to complete the task.
 
+### Truncation with tooltip
+
+To display the full text when truncatioon occurs, hover over the truncated text and a tooltip will be shown with the full text.
+
+Tooltip presents information about a specific element in a small conntainer on top of other content upon hovering. 
```

Comment:
```suggestion
Upon hovering, Tooltip presents additional information about a specific element in a small container. The Tooltip will remain visible as long as the user is hovering over the element.
```

Reply:
Since "Tooltip" isn't the name of the component, I think it should be referred with a lower case ("tooltip"). for referring the Tip component, it will make sense to use Tip.